hbp_validation_framework/__init__.py

```diff
@@ -1761,10 +1761,10 @@ def list_model_instances(self, instance_path="", model_id="", alias=""):
         model_instances_json = model_instances_json.json()
         return model_instances_json["instances"]
 
-    def add_model_instance(self, model_id="", alias="", source="", version="", parameters=""):
+    def add_model_instance(self, model_id="", alias="", source="", version="", description="", parameters=""):
         """Register a new model instance.
 
-        This allows to add a new instance of an existing model in the model catalog.
+        This allows to add a new instance(s) of an existing model in the model catalog.
```

Comment:
I don't understand how this can be used to add multiple instances.

Reply:
My mistake. The POST request itself is capable of adding multiple instances in one go, but the implementation in the client (intentionally) does not permit that. Will undo this change in the docs.